### ISSUE TITLE
select: toggle and option target sizes

### ISSUE THREAD

**Issue by nikkimk** [patternfly/patternfly-elements#2665]
## Description of the issue

Select target sizes should be a 24px minimum, or  [WCAG 2.2 2.5.8 AA compliant](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#258-target-size-minimum-aa).

### Impacted component(s)

- `pf-select`
- `pf-option`

### Steps to reproduce

1. Go to [select documentation](https://patternflyelements.org/components/select/)
2. Inspect the toggle button.
3. Inspect the options.
4. Inspect the typeahead text field.

### Expected behavior

Minimum width and height should be 24px.

**Comment by bennypowers** [comment]
we can't diverge from upstream styles. a11y issues on patternfly should be filed there

**Comment by bennypowers** [comment]
@nikkimk can we close this one?
